backend/routes/camera.py

```diff
@@ -8,3 +8,12 @@
 def home():
     return "This is default Camera Page"
 
+@camera_page.route('/addCamera', methods=["POST", "GET"])
+def add_camera():
```

Comment:
After we had API , decide what data format we need from UI (json, form data, ...).